setup.cfg

```diff
@@ -1,3 +1,25 @@
+[metadata]
+name = pickLayer
+version = attr: pickLayer.__version__
```

Comment:
this must match the location where to find the property, so `pickLayer.core.__version__` if it is defined in core package.

it actually currently works, since the `setup.py` file defines a version number in the arguments, so this one is not used. `setup.py` should be changed to call `setup()` with no args, so that this `setup.cfg` is used instead.